BO4E/ENUM/Handelsunstimmigkeitsgrund.cs

```diff
@@ -31,5 +31,41 @@ public enum Handelsunstimmigkeitsgrund
         /// <summary> SONSTIGES_SIEHE_BEGRUENDUNG </summary>
         /// <remarks>28</remarks>
         SONSTIGES_SIEHE_BEGRUENDUNG,
+
+        /// <summary> GUELTIGES_PREISBLATT_VERSENDET </summary>
+        /// <remarks>A01</remarks>
+        GUELTIGES_PREISBLATT_VERSENDET,
+
+        /// <summary> GUELTIGER_SPERRAUFTRAG_VORHANDEN </summary>
+        /// <remarks>A02</remarks>
+        GUELTIGER_SPERRAUFTRAG_VORHANDEN,
+
+        /// <summary> KORREKTE_ARTIKEL_ID_IN_RECHNUNG </summary>
+        /// <remarks>A03</remarks>
+        KORREKTE_ARTIKEL_ID_IN_RECHNUNG,
+
+        /// <summary> KORREKTER_PREIS_ZU_GUELTIGEM_PREISBLATT_IN_RECHNUNG </summary>
+        /// <remarks>A05</remarks>
+        KORREKTER_PREIS_ZU_GUELTIGEM_PREISBLATT_IN_RECHNUNG,
```

Comment:
Bist du hier mit A04 und A05 durcheinander gekommen?

Reply:
Nice catch, danke dir! ja ich brauch nur A04, A05 gibt in meinem Fall gar nicht